code/env/prod/terraform.tfvars

```diff
@@ -30,24 +30,25 @@ cidr_subnet_redis         = ["10.0.8.0/24"]
 cidr_subnet_vpn           = ["10.0.133.0/24"]
 cidr_subnet_dns_forwarder = ["10.0.134.0/29"]
 
-database_name            = "cgnonboardingportal"
-db_sku_name              = "GP_Gen5_8"
-db_version               = 11
-devops_admin_email       = "io-operations@pagopa.it"
-dns_zone_prefix          = "cgnonboardingportal"
-email_department_email   = "cartagiovaninazionale@governo.it"
-enable_ade_aa_mock       = true
-enable_custom_dns        = true
-enable_spid_test         = false
-env_short                = "p"
-external_domain          = "pagopa.it"
-io_apim_resourcegroup    = "io-p-rg-internal"
-io_apim_name             = "io-p-apim-api"
-io_apim_productid        = "/subscriptions/ec285037-c673-4f58-b594-d7c480da4e8b/resourceGroups/io-p-rg-internal/providers/Microsoft.ApiManagement/service/io-p-apim-api/products/cgnmerchant"
-io_apim_subscription_id  = "ec285037-c673-4f58-b594-d7c480da4e8b"
-redis_cache_family       = "C"
-redis_cache_sku_name     = "Standard"
-operator_search_capacity = 1
+database_name             = "cgnonboardingportal"
+activations_database_name = "cgnonboardingportal-activations"
```

Comment:
why not attribute-authority?

Reply:
Right! It's better.

Reply:
fixed in https://github.com/pagopa/cgn-onboarding-portal-infra/pull/82/commits/9d084f39c33fbfe0b2f6bbc7123b656e925b19ca